src/ucp/core/ucp_worker.h

```diff
@@ -338,6 +338,10 @@ typedef struct ucp_worker {
                                                            * used to minimize call of ucs_get_time */
         size_t                       round_count;         /* Number of rounds done */
     } keepalive;
+    /* Number of requests to create endpoint */
+    uint64_t                         num_ep_creations;
+    /* Number of failed requests to create endpoint */
+    uint64_t                         num_ep_creation_failures;
```

Comment:
maybe separate to sub_struct
struct {
  uint64_t ep_create;
   uint64_t ep_create_failure;
 } counters;